packages/frontend/src/components/accounts/ledger/SetupLedger.js

```diff
@@ -93,12 +106,15 @@ const SetupLedger = (props) => {
                             return;
                         }
 
-                        await dispatch(createNewAccount({ accountId, fundingOptions, recoveryMethod: 'ledger', publicKey, recaptchaToken })).unwrap();
+                        await dispatch(createNewAccount({ accountId, fundingOptions, recoveryMethod: 'ledger', publicKey, recaptchaToken, path: customLedgerHdPath, handleCloseModal:()=>{
+                            dispatch(checkAndHideLedgerModal());
```

Comment:
I'm not sure about this solution, where we pass a function that is dispatching an action as a param to another action that is being dispatched. 

If you want to close the modal manual, you can just dispatch `checkAndHideLedgerModal` action inside `createNewAccount` thunk.

But we have an implemented solution that is closing the modal automatically, and it seems like something is broken there. If you would like to investigate it, let me know and I can tell you where to look, or you can assign me to the issue and I will handle it.